tests/test_utils.py

```diff
@@ -14,3 +16,36 @@ def test_cleanse_text_without_strings():
     assert cleanse_text(None) is None
     assert cleanse_text(1) == 1
     assert cleanse_text({}) == {}
+
+
+def test_get_out_spatial_reference_get_request():
+    request = MagicMock()
+    request.method = "GET"
+    request.args = {"outSR": 3857}
+
+    assert get_out_spatial_reference(request) == (3857, 3857)
+
+
+def test_get_out_spatial_reference_get_request_json():
+    request = MagicMock()
+    request.method = "GET"
+    request.args = {"outSR": '{"wkid": 3857}'}
```

Comment:
**question**: should we test for and use the value of `lastestWkid` if it's available?

Reply:
punted to #180